cmd/swarm/main.go

```diff
@@ -181,6 +181,18 @@ var (
 		Usage:  "Number of recent chunks cached in memory (default 5000)",
 		EnvVar: SWARM_ENV_STORE_CACHE_CAPACITY,
 	}
+	SwarmResourceMultihashFlag = cli.BoolFlag{
```

Comment:
@nolash please, save me. We're going in circles here...
;-)

Reply:
@zelig I just asked @jpeletier to change this back because there was no good flag name proposed (`--multihash=false` with multihash as default was the prevailing one,that's clumsy). According to rumors, `--raw` could not be used. `--rawmru` stinks IMO. Do you have a good name?

Personally, I think multihash should be the exception and not the other way around.